run_dir/static/js/data_del_plot.js

```diff
@@ -0,0 +1,357 @@
+window.current_plot_data = null;
+window.current_plot_obj = null;
+window.current_view_schemes = null;
+
+$(function(){
+    init_page_js();
+});
+
+function refresh_plot(){
+    var params = get_parameters();
+    if (window.current_plot_data == null){
+        get_plot_data(key=params[0], name=params[1], view_type=params[2], search_string=params[3], inst_type_filter=params[4]);
+    }else{
+        make_plot(key=params[0], name=params[1], view_type=params[2], inst_type_filter=params[4]);
+    }
+}
+
+function make_plot(key, name, view_type, filter_inst_type){
+    var toplot = {
+        chart: {
+            type: 'scatter',
+            events: {
+                render: function(){
```

Comment:
This seems like a slightly complicated way to do things. You have a function to make the plot. Then inside the plot config, you're having another function to edit the title after the plot is generated to dynamically update the title.

Instead, I would calculate this _before_ the `var toplot` object is created and just set the title with the variable in the normal way (`text : name+' of projects '+total_filesize` or whatever)